zones/absurdopedia.wiki

```diff
@@ -0,0 +1,31 @@
+$TTL 300
+$ORIGIN absurdopedia.wiki.
+
+@   SOA ns1.miraheze.org. hostmaster.miraheze.org. (
+		20190206000001	; serial
```

Comment:
Should be 20190301000001

Reply:
I never know how to get the proper value for this

Reply:
<date>000001